wire-library/wire-grpc-server-generator/src/test/golden/MethodDescriptor.kt

```diff
@@ -0,0 +1,32 @@
+package routeguide
+
+import io.grpc.MethodDescriptor
+import kotlin.jvm.Volatile
+
+public class RouteGuideWireGrpc {
+  @Volatile
+  private var getGetFeatureMethod: MethodDescriptor<Point, Feature>? = null
```

Comment:
consider using Kotlin’s `by lazy` instead of manual double checked locking?

Reply:
This and the get with `@Volatile` are copies of the generated plain Java gRPC code. I'm not sure of the implications of using `by lazy` that might make it different from this (though I think it should be the same).

Reply:
When in Kotlin . . .